configs/mender_convert_config

```diff
@@ -4,12 +4,18 @@
 #
 # NOTE! This file will always be sourced.
 
-# Compress generated disk image using gzip
+# Compress generated disk image
 #
 # This is useful when you have large disk images, compressing them
 # makes it easier to transfer them between e.g an build server and a local
 # machine, and obviously saves space.
-MENDER_COMPRESS_DISK_IMAGE=y
+#
+# Possible values are:
+# 'n'    - No compression
+# 'gzip' - Compress with gzip
+# 'lzma' - Compress with xz (LZMA)
+# 'y'    - Alias for 'gzip'
+MENDER_COMPRESS_DISK_IMAGE=gzip
```

Comment:
Do not know if we need backwards compatibility here as we have not released this yet, meaning maybe we can drop `y/n` and keep the interface similar to the `MENDER_ARTIFACT_COMPRESSION`

In case we drop `y` we need to update https://github.com/mendersoftware/mender-convert/blob/next/scripts/run-tests.sh

Reply:
Good point, will fix this!